sdk/core/core-http/src/nodeFetchHttpClient.ts

```diff
@@ -111,6 +111,8 @@ export class NodeFetchHttpClient extends FetchHttpClient {
     // Set the http(s) agent
     requestInit.agent = this.getOrCreateAgent(httpRequest);
 
+    requestInit.compress = httpRequest.compress;
```

Comment:
Is this a setting that's used by node-fetch? Can you clarify what this actually does? I'm expecting that it updates the `Accepts` header to specify it can accept gzip content, and that node-fetch is also decompressing the content as it comes back from the response when necessary.

Is this correct? Calling the option `compress` makes me think we're going to compress whatever request we're sending.

Reply:
https://github.com/node-fetch/node-fetch#options

It updates Accept-Encoding to gzip,deflate if options.compress === true, which is default.  

> Calling the option compress makes me think we're going to compress whatever request we're sending.
Agree. I had the same feeling as well, but haven't thought of a better name yet.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding